### DISCUSSION TITLE
Alias' Title Cased?

### DISCUSSION THREAD

**Discussion by josephgruber** [pydantic/pydantic#5679] [Question]
When providing an alias, the schema generation then modifies the format of that alias to be title cased and replaces underscores with spaces. See `get_field_info_schema` in schema.py:

```
def get_field_info_schema(field: ModelField) -> Tuple[Dict[str, Any], bool]:
    schema_overrides = False

    # If no title is explicitly set, we don't set title in the schema for enums.
    # The behaviour is the same as `BaseModel` reference, where the default title
    # is in the definitions part of the schema.
    schema: Dict[str, Any] = {}
    if field.field_info.title or not lenient_issubclass(field.type_, Enum):
        schema['title'] = field.field_info.title or field.alias.title().replace('_', ' ')
```

Is there a particular reason that Pydantic decideds to title case the alias going around the user's formatting desires? Would like to understand if there's a specific reason for this before I submit a bug.

Additionally, this if statement will evaluate True to basically anything other than an Enum (if title or not enum).

**Comment by dmontagu** [comment]
I think we aren't likely to modify this behavior in v1 since we are treating v1 as in "maintenance mode", and are hesitant to introduce any new changes to v1 that might lead to debate about the ideal behavior.

However, for what it's worth, I think this should be much easier to override in v2 due to the way we have replaced the slew of schema-generation functions with a class that has somewhat granular methods available to be overridden, and you can use your custom subclass when producing JSON schema.
